pkg/routes/route.go

```diff
@@ -0,0 +1,103 @@
+// Copyright (c) Mainflux
+// SPDX-License-Identifier: Apache-2.0
+
+package routes
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mainflux/export/pkg/messages"
+	"github.com/mainflux/mainflux/logger"
+	nats "github.com/nats-io/nats.go"
+)
+
+// Route - message route, tells which nats topic messages goes to which mqtt topic.
```

Comment:
What is this comment about? The file? If yes - put it on the top of the file, after the copyright header.